app/src/main/java/com/example/playlistmaker/player/ui/PlayerState.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.playlistmaker.player.ui
+
+sealed interface PlayerState {
+    object Default: PlayerState
+    object Prepared: PlayerState
+    data class Playing(var time: String): PlayerState
+    data class Paused(var time: String): PlayerState
```

Comment:
```suggestion
    data class Playing(val time: String): PlayerState
    data class Paused(val time: String): PlayerState
```
Вроде не должно быть кейсов, когда подписчик на этот стейт должен менять эту переменную, поэтому val